lib/mage/Mage.js

```diff
@@ -39,6 +39,9 @@ function testEngineVersion(pkg, pkgPath, mage) {
  */
 
 function Mage(mageRootModule) {
+	// Set up the worker id, if necessary
+	this.workerId = require('./worker').getId();
```

Comment:
Maybe do this after the `EventEmitter.call(this);`?

Reply:
Done.